staging/src/k8s.io/api/extensions/v1beta1/types.go

```diff
@@ -683,7 +683,25 @@ type IngressTLS struct {
 type IngressStatus struct {
 	// LoadBalancer contains the current status of the load-balancer.
 	// +optional
-	LoadBalancer v1.LoadBalancerStatus `json:"loadBalancer,omitempty" protobuf:"bytes,1,opt,name=loadBalancer"`
+	LoadBalancer IngressLoadBalancerStatus `json:"loadBalancer,omitempty" protobuf:"bytes,1,opt,name=loadBalancer"`
+}
+
+// LoadBalancerStatus represents the status of a load-balancer.
+type IngressLoadBalancerStatus struct {
```

Comment:
the beta types too?

Reply:
actually the failing tests are the related to compatibility
```
k8s.io/kubernetes/vendor/k8s.io/api: TestCompatibility/extensions.v1beta1.Ingress/HEAD expand_more
```
let me summon Jordan @liggitt 


<br class="Apple-interchange-newline">

Reply:
The tests are failing because round-tripping fixture data serialized using 1.21, 1.22, and master versions of the ingress types through the types in this PR are dropping data:

```
=== RUN   TestCompatibility/networking.k8s.io.v1.Ingress/HEAD
    compatibility.go:303: json differs
    compatibility.go:304:   (
          	"""
          	... // 98 identical lines
          	        {
          	          "ip": "35",
        - 	          "hostname": "36",
        - 	          "ports": [
        - 	            {
        - 	              "port": -8441701,
        - 	              "protocol": "",
        - 	              "error": "37"
        - 	            }
        - 	          ]
        + 	          "hostname": "36"
          	        }
          	      ]
          	... // 3 identical lines
          	"""
          )
        
    compatibility.go:309: yaml differs
    compatibility.go:310:   (
          	"""
          	... // 67 identical lines
          	    - hostname: "36"
          	      ip: "35"
        - 	      ports:
        - 	      - error: "37"
        - 	        port: -8441701
        - 	        protocol: ""
          	"""
          )
        
    compatibility.go:315: proto differs
    compatibility.go:317: exec: "protoc": executable file not found in $PATH
    compatibility.go:317: exec: "protoc": executable file not found in $PATH
    compatibility.go:317: 
    compatibility.go:330: if the diff is expected because of a new type or a new field, re-run with UPDATE_COMPATIBILITY_FIXTURE_DATA=true to update the compatibility data
        --- FAIL: TestCompatibility/networking.k8s.io.v1.Ingress/HEAD (0.01s)
```

In this case, this is expected because you dropped a field. Running the unit test manually with `UPDATE_COMPATIBILITY_FIXTURE_DATA=true` per the instructions will update HEAD fixtures and generated `...after_roundtrip` updates for the 1.21 and 1.22 fixtures